lib/bundler/cli.rb

```diff
@@ -333,7 +333,8 @@ def binstubs(*gems)
     method_option "version", :aliases => "-v", :type => :string
     method_option "group", :aliases => "-g", :type => :string
     method_option "source", :aliases => "-s", :type => :string
-
+    method_option "skip-install", :type => :boolean, :banner =>
+      "Adds gem to gemfile but does not install it"
```

Comment:
Cool addition!  :hearts: :yellow_heart: :green_heart: :blue_heart: :purple_heart:

Suggestion: harmonize wording with the rest of the options: `to gemfile` => `to the Gemfile`. (In this file and and in the .ronn documentation.)